test.ts

```diff
@@ -447,6 +447,18 @@ test(function invalidBindDoesNotLeakStatements() {
   db.close();
 });
 
+test(function getColumnsFromRows() {
+  const db = new DB();
+
+  db.query("CREATE TABLE test (id INTEGER, name TEXT)");
+  db.query("INSERT INTO test (id, name) VALUES (?, ?)", [1, "name"]);
```

Comment:
Nit: If you make id a `INTEGER PRIMARY KEY AUTOINCREMENT`, you won't have to set them here. But this is fine too (I think it might actually be shorter, as you only add one key 😅)